proto/schema/v1/schema.proto

```diff
@@ -4,6 +4,9 @@ package sonrio.sonr.schema;
 
 option go_package = "github.com/sonr-io/sonr/x/schema/types";
 
+option java_package = "io.sonr.types.schema";
```

Comment:
Should this be `"io.sonr.schemas.types"`

Reply:
The project structure in Java doesn't separate out the modules like that, it has:
* io.sonr.motoroid
and that's it

So rather than creating a package for each domain that contains only a `types` package, it makes more sense to create one `types` package which contains all the domains